sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/utils/mock-utils.ts

```diff
@@ -0,0 +1,20 @@
+import { DeepMocked, createMock } from "@golevelup/ts-jest";
+import { getISODateOnlyString } from "@sims/utilities";
+import { Job } from "bull";
+import { GeneratedDateQueueInDTO } from "../../../models/ier.model";
+
+/**
+ * Creates the mocked IER12 job payload for the scheduler.
+ * @param date date of processing.
+ * @returns mocked IER12 payload.
+ */
+export function createIER12SchedulerJobMock(
+  date: string | Date,
```

Comment:
date must be an optional parameter in the mock right? (thinking close to the real time)

Reply:
I will change it, right now it is not needed because all TCs are providing the dates.

Reply:
Changed to accept the date as optional.